apps/frontend/src/libs/components/balance-wheel/libs/helpers/generate-gradient.helper.ts

```diff
@@ -0,0 +1,41 @@
+import { type ScriptableContext } from "chart.js";
```

Comment:
We have imports from `chart.js` in too many files, export all types from one file

Reply:
Should I create a chart.js export file globally or just inside the balance wheel component?

Reply:
Let's make it global, we don't want to exclude the possibility of adding other charts

Reply:
I have another question. Should I also export non-type chart elements in the same file as the chart types?

```
export {
	ArcElement,
	Chart as ChartJS,
	type ChartConfiguration,
	PolarAreaController,
	RadialLinearScale,
	type ScriptableContext,
	Tooltip,
} from "chart.js";
```

Reply:
You can import chart elements directly in chart component file. But export types from ~/libs/types/types